client/executor/common/src/sandbox.rs

```diff
@@ -56,10 +53,10 @@ impl From<SupervisorFuncIndex> for usize {
 ///
 /// This index is supposed to be used as index for `Externals`.
 #[derive(Copy, Clone, Debug, PartialEq)]
-struct GuestFuncIndex(usize);
+pub(crate) struct GuestFuncIndex(pub(crate) usize);
```

Comment:
`pub (crate)` is usually a sign of bad structuring of the modules: The backends should be a sub module of this.

Reply:
Absolutely. It was just a quick hack during the refactiring.